src/utils/miscellaneous.ts

```diff
@@ -5,8 +5,8 @@ function noop(..._args: any[]): void {}
 
 export const log = process.env.NODE_ENV === 'test' ? noop : console.log
 
-export function getToken<T extends TokenDetails, K extends keyof T>(
-  key: K,
+export function getToken<T extends TokenDetails>(
+  key: string,
```

Comment:
Why change this back?

Reply:
when is the key of TokenDetails not a string? it made the line below, `value.toString()` hard to type

Reply:
The change was not because the key isn't string, but because the typing enforces the parameters will always be existing keys in TokenDetails. See for reference https://mariusschulz.com/blog/keyof-and-lookup-types-in-typescript#indexed-access-types. This is where I found out about it.